src/sentry/sentry_metrics/indexer/models.py

```diff
@@ -50,3 +56,76 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=["string", "organization_id"], name="unique_org_string"),
         ]
+
+
+class StringIndexerCache:
+    def __init__(self, model: Any, version: int):
+        self.model = model
+        self.version = version
+        self.cache_ttl: int = settings.SENTRY_METRICS_INDEXER_CACHE_TTL
+
+    @property
+    def randomized_ttl(self) -> int:
+        # introduce jitter in the cache_ttl so that when we have large
+        # amount of new keys written into the cache, they don't expire all at once
+        jitter = random.uniform(0, 0.25) * self.cache_ttl
```

Comment:
What is the usual `cache_ttl`? 25% jitter seems really high.

Reply:
the current indexer `cache_ttl` is 2hrs, so I think maybe 25% is okay for now?